pkg/apis/submariner.io/v1/types.go

```diff
@@ -77,10 +79,20 @@ type GatewayStatus struct {
 	Version       string       `json:"version"`
 	HAStatus      HAStatus     `json:"haStatus"`
 	LocalEndpoint EndpointSpec `json:"localEndpoint"`
+	// +optional
+	Latency       LatencySpec  `json:"latency"`
 	StatusFailure string       `json:"statusFailure"`
 	Connections   []Connection `json:"connections"`
 }
 
+type LatencySpec struct {
+	Last string `json:"last"`
+	Min  string `json:"min"`
+	Avg  string `json:"avg"`
+	Max  string `json:"max"`
+	Mdev string `json:"mdev"`
```

Comment:
Does the golang package we plan to use for "ping" provide all this info?

Reply:
I need to check this. Will update.

Reply:
It seems to be supported in one of the libraries [1] I checked.

[1]https://godoc.org/github.com/sparrc/go-ping